consensus/consensus_tests/utils_test.go

```diff
@@ -176,21 +181,21 @@ func StartAllTestPocketNodes(t *testing.T, pocketNodes IdToNodeMapping) {
 // TODO(discuss): Should we use reflections inside the testing module as being done here or explicitly
 // define the interfaces used for debug/development. The latter will probably scale more but will
 // require more effort and pollute the source code with debugging information.
-func GetConsensusNodeState(node *shared.Node) typesCons.ConsensusNodeState {
+func GetConsensusNodeState(node modules.NodeModule) typesCons.ConsensusNodeState {
```

Comment:
I don't think this change in naming convention made it clearer (I think the opposite)

Reply:
I am not sure I understand what you mean. 

Is the fact that we are accepting the interface or the name of the interface that sounds odd?